packages/suite/src/utils/suite/homescreen.ts

```diff
@@ -186,48 +186,65 @@ export const elementToImageData = (element: HTMLImageElement, width: number, hei
     return imageData;
 };
 
-export const checkImage = (origImage: HTMLImageElement, model: number) => {
+export enum ImageValidationError {
```

Comment:
you can use `export const enum` it will produce little bit better compiled output and maybe intellisense will be happier

Reply:
b3c29f109b8271f64475a9bcf20e13d2a7352bac